client/me/security-2fa-initial-setup/index.jsx

```diff
@@ -35,10 +48,54 @@ class Security2faInitialSetup extends Component {
 					) }
 				</p>
 
+				<FormFieldset>
+					<FormRadio
+						id="app-auth"
+						name="auth_method"
+						value="app-based"
+						defaultChecked
+						onChange={ this.setAuth }
+					/>
+					<FormLabel htmlFor="app-auth">
+						<Gridicon icon="phone" />
+						<span className="security-2fa-initial-setup__item-title">
+							{ this.props.translate( 'Set up using an app', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?